ci/Jenkinsfile

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.7.13'
+
+pipeline {
+    agent {
+        label 'linux && x86_64 && qt-5.15.2'
+    }
+
+    environment {
+        SQUISH_DIR = "/opt/squish-runner-7.1-20230222-1555"
+        PYTHONPATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/lib/python:${PYTHONPATH}"
+        LD_LIBRARY_PATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/python3/lib:${LD_LIBRARY_PATH}"
+    }
+
+    parameters {
+        string(
+          name: 'BRANCH',
+          description: 'Paste a branch name from test repository.',
+          defaultValue: 'master'
+        )
+        string(
+          name: 'BUILD',
+          description: 'Paste a number of build that you want to test.',
+          defaultValue: 'PR-'
+        )
+        string(
+          name: 'FILTER',
+          description: 'Paste "-k" with test name/part of test name to run specific test or/and "-m" with tag to run specific scope of tests',
+          defaultValue: ''
+        )
+
+        string(
+          name: 'TEST_RUN',
+          description: 'Test run ID in Test Rail',
+          defaultValue: ''
+        )
+    }
+
+    options {
+        timestamps()
+        /* Prevent Jenkins jobs from running forever */
+        timeout(time: 120, unit: 'MINUTES')
+        /* manage how many builds we keep */
+        buildDiscarder(logRotator(
+            numToKeepStr: '10',
+            daysToKeepStr: '30',
+            artifactNumToKeepStr: '3',
+        ))
+    }
+
+    stages {
+        stage('Clean') {
+            steps {
+                sh "rm -rf ${env.WORKSPACE}/tmp/"
+                sh "rm -rf ${env.WORKSPACE}/allure-results"
+            }
+        }
+
+        stage('Download') {
+            steps { script {
+                copyArtifacts(
+                    projectName: "status-desktop/branches/linux/x86_64/package/${params.BUILD}/",
+                    filter: 'pkg/*-x86_64.tar.gz',
+                    selector: lastWithArtifacts(),
+                    target: "${env.WORKSPACE}"
+                )
+                pkg_path = "${env.WORKSPACE}/${utils.findFile('pkg/*-x86_64.tar.gz')}"
```

Comment:
Please use `def` for variables that should be in local scope:
https://www.baeldung.com/groovy/variable-scope

Reply:
Also, searching the whole `WORKSPACE` is a waste of time and I/O. Why not just copy the artifact to it's own dedicated folder?

Reply:
Fixed